core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -196,6 +201,49 @@ private[spark] class SecurityManager(sparkConf: SparkConf) extends Logging with
     )
   }
 
+  private[spark] val sslOptions = SSLOptions.parse(sparkConf, "spark.ssl")
```

Comment:
The `SecurityManager` class itself is `private[spark]`, so I think methods for Spark internal use can just be public.